testing/SPEC-CONFORMANCE.md

```diff
@@ -130,8 +130,10 @@ The conformance requirements for EBU-TT Part 3 derive from the specification its
 ||| **Tech 3370 v1.0**||
-|R120|2.4.2|The Handover Manager shall maintain all of the following variables: a token equal to the value of the `ebuttp:authorsGroupControlToken` of the most recently emitted document, or a null value if no document has been emitted; a record of the sequence identifier of the source sequence used to generate the most recently emitted document, known as the selected sequence, initially a null value until a sequence is selected; a record of the configured authors group identifier; an output sequence identifier.|Tested implicitly as part of the algorithm in `bdd/features/handover/handover_algorithm.feature`|
-|R121|3.2.2.1|The data type of `ebuttp:authorsGroupControlToken` shall be a positive integer.|`xsd/ebutt_parameters.xsd`|
-|R122|3.2.2.1|The data type of `ebuttp:sequenceNumber` shall be a positive integer.|`xsd/ebutt_parameters.xsd`|
-|R123|3.2.2.1|The data type of `ebuttp:sequenceIdentifier` shall be a string of length >= 1.|`bdd/features/validation/sequence\_id\_num.feature` `(In)valid Sequence head attributes`|
-|R124|3.2.2.1|The data type of `ebuttp:authorsGroupIdentifier` shall be a string of length >= 1.|`bdd/features/handover/handover.feature`|
\ No newline at end of file
+|120|2.4.2|Valid input for the handover manager is a valid document that contains all of: 1) a `ebuttp:authorsGroupIdentifier` equal to the configured authors group identifier; and 2) an `ebuttp:authorsGroupControlToken`|`bdd/features/handover/handover.feature`|
+|R121|2.4.1|The Handover manager shall include within each output document the metadata attribute `authorsGroupSelectedSequenceIdentifier` set to the value of the source sequence's identifier for that document.|`bdd/features/handover/handover_algorithm.feature`|
+|R122|3.2.2.1|The data type of `ebuttp:authorsGroupControlToken` shall be a positive integer.|`xsd/ebutt_parameters.xsd`|
+|R123|3.2.2.1|The data type of `ebuttp:sequenceNumber` shall be a positive integer.|`xsd/ebutt_parameters.xsd`|
+|R124|3.2.2.1|The data type of `ebuttp:sequenceIdentifier` shall be a string of length >= 1.|`bdd/features/validation/sequence\_id\_num.feature` `(In)valid Sequence head attributes`|
+|R125|3.2.2.1|The data type of `ebuttp:authorsGroupIdentifier` shall be a string of length >= 1.|`bdd/features/handover/handover.feature`|
+|R126|2.4.2|The Handover Manager shall maintain all of the following variables: a token equal to the value of the `ebuttp:authorsGroupControlToken` of the most recently emitted document, or a null value if no document has been emitted; a record of the sequence identifier of the source sequence used to generate the most recently emitted document, known as the selected sequence, initially a null value until a sequence is selected; a record of the configured authors group identifier; an output sequence identifier.|Tested implicitly as part of the algorithm in `bdd/features/handover/handover_algorithm.feature`|
```

Comment:
§2.4.2 has a 'shall' before the algorithm part too, so the things in that algorithm need to be captured - R121 for example is one part of this already.